Cartfile.resolved

```diff
@@ -4,7 +4,7 @@ github "Alamofire/AlamofireImage" "c41f8b0acfbb3180fe045df73596e4332c338633"
 github "ReactiveCocoa/ReactiveSwift" "6.0.0"
 github "facebook/facebook-objc-sdk" "v5.0.2"
 github "kickstarter/Kickstarter-Prelude" "d03f8831dcec9ad157db4f1c5f769ec17e1a3766"
-github "kickstarter/Kickstarter-ReactiveExtensions" "aa994f0921dca1965ae537fe77c2edc798eef230"
+github "kickstarter/Kickstarter-ReactiveExtensions" "99c3013e22fd6d4ea29d9d4c7a4a7b10bf80a99c"
```

Comment:
It's weird that it builds on Circle and not on my machine. I wonder what's the real problem here. 🤔

Reply:
The hash in this file should match the one in the `Cartfile`.